src/file_pipe_log/mod.rs

```diff
@@ -47,7 +47,7 @@ pub mod debug {
             file_system.open(path)?
         };
         let fd = Arc::new(fd);
-        super::log_file::build_file_writer(file_system, fd, version)
+        super::log_file::build_file_writer(file_system, fd, version, false)
```

Comment:
Same for other uses.
```suggestion
        super::log_file::build_file_writer(file_system, fd, version, false /*force_reset*/)
```